x-pack/plugins/security_solution/common/api/detection_engine/rule_monitoring/rule_execution_logs/get_rule_execution_events/get_rule_execution_events_route.schema.yaml

```diff
@@ -14,47 +14,47 @@ paths:
         - name: ruleId
           in: path
           required: true
-          description: Saved object ID of the rule to get execution results for
           schema:
             type: string
             minLength: 1
-        - name: start
+        - name: search_term
           in: query
-          required: true
-          description: Start date of the time range to query
-          schema:
-            type: string
-            format: date-time
-        - name: end
-          in: query
-          required: true
-          description: End date of the time range to query
+          required: false
+          description: Include events of matching the search term. If omitted, all events will be included.
           schema:
             type: string
-            format: date-time
-        - name: query_text
+        - name: event_types
           in: query
           required: false
-          description: Query text to filter results by
+          description: Include events of the specified types. If omitted, all types of events will be included.
           schema:
-            type: string
-            default: ''
-        - name: status_filters
+            type: array
+            items:
+              $ref: '../../model/execution_event.schema.yaml#/components/schemas/RuleExecutionEventType'
+            default: []
+        - name: log_levels
           in: query
           required: false
-          description: Comma-separated list of rule execution statuses to filter results by
+          description: Include events having these log levels. If omitted, events of all levels will be included.
           schema:
             type: array
             items:
-              $ref: '../../model/execution_status.schema.yaml#/components/schemas/RuleExecutionStatus'
+              $ref: '../../model/execution_event.schema.yaml#/components/schemas/LogLevel'
             default: []
-        - name: sort_field
+        - name: date_start
           in: query
           required: false
-          description: Field to sort results by
+          description: Start date of the time range to query
           schema:
-            $ref: '../../model/execution_result.schema.yaml#/components/schemas/SortFieldOfRuleExecutionResult'
-            default: timestamp
+            type: string
+            format: date-time
+        - name: date_end
+          in: query
+          required: false
+          description: End date of the time range to query
+          schema:
+            type: string
+            format: date-time
```

Comment:
Not modified in this PR, but I see that the parms `sort_order`, `page` and `per_page` are marked as required: false, while in `GetRuleExecutionEventsRequestQuery` in the deleted `x-pack/plugins/security_solution/common/api/detection_engine/rule_monitoring/rule_execution_logs/get_rule_execution_events/get_rule_execution_events_route.ts` they are not marked as `partial`. 

I realize that these three fields have a default value here in the schema, but not sure if that is enough for the query to pass validation in run-time.

Reply:
Yeah, it looks like a mistake in the previous schema. Defaultable fields should not be required